sourcemash/frontend/static/js/default.js

```diff
@@ -14,4 +16,12 @@ $(document).ajaxError(function (e, xhr, options) {
         fieldErrors.text(errors[errorField]);
     }
   }
-});
\ No newline at end of file
+
+  // Show Modal, if Not Authenticated
+  var errorStatus = xhr.status;
+  if (errorStatus == 401) {
```

Comment:
No point to save errorStatus as a var when we just use it once. Want to make it ```if (xhr.status == 401)```?